server/controllers/entities/lib/properties/properties_values_constraints.coffee

```diff
@@ -65,6 +73,8 @@ module.exports =
   'wdt:P569': bases.uniqueSimpleDay
   # date of death
   'wdt:P570': bases.uniqueSimpleDay
+  # inception
+  'wdt:P571': bases.uniqueSimpleDay
```

Comment:
it looks like fae5f7c mistakenly removed `wdt:P576` or am I missing something?

Reply:
not really mistakenly, i decided it wasnt a relevant information enough to be displayed so i deleted without explaining my motivation...

Reply:
[for documentation purpose] could actually be useful for check publication editions from a post-mortem editor